packages/appstore/src/components/options/options.scss

```diff
@@ -0,0 +1,171 @@
+.options-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1.5rem;
+    width: 115.2rem;
+    height: 30.3rem;
+    @include mobile {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        position: absolute;
+        width: 32.8rem;
+        height: 28rem;
+    }
+
+    &__title-description-container {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        width: 115.2rem;
```

Comment:
You already have the width set in your parent. what is the reason of having width over here ?
You can use width: something% here which will work well according to your parents width.
Having so many static values is not a recommended way of coding.